podcasts/Onboarding/Plus/PlusPurchaseModal.swift

```diff
@@ -12,35 +12,41 @@ struct PlusPurchaseModal: View {
         coordinator.pricingInfo
     }
 
-
     /// Whether or not all products have free trials, in this case we'll show the free trial label
     /// above the products and not inline
     let showGlobalTrial: Bool
-    init(coordinator: PlusPurchaseModel) {
+
+    private var planToPurchase: Constants.Plan
+
+    private var products: [PlusPricingInfoModel.PlusProductPricingInfo]
+
+    init(coordinator: PlusPurchaseModel, planToPurchase: Constants.Plan, selectedPrice: UpgradeLandingView.DisplayPrice = .yearly) {
```

Comment:
Since the `PlusPurchaseModel` is storing the `plan` wdyt about removing the planToPurchase?

Reply:
Done in c4c98dcf80d75f422f59386613d4ddc27b433706